core/domain/exp_domain.py

```diff
@@ -4223,7 +4247,10 @@ def validate(self):
                 'Assigned state ids should be smaller than last state id used.')
 
         if not all(isinstance(x, int) for x in state_ids):
-            raise Exception('Assigned state ids should be integer values.')
+            raise Exception(
+                'Assigned state ids should be integer values in'
+                ' exploration %s, version %d' % (
```

Comment:
Nit: space that prefixes this line should go at the end of the previous line, instead.

Reply:
Done